dali/base/dautils.cpp

```diff
@@ -515,7 +523,17 @@ void CDfsLogicalFileName::set(const char *name, bool removeForeign)
         lfn.set(full);
         return;
     }
-    normalizeName(name, lfn, false);
+    if (isExternalFile(name))
+    {
+        lfn.set(name);
+        lfn.toLowerCase();
+        const char *ns1 = strstr(name,"::>");
```

Comment:
I'm not sure if it's really used (it was something to do with RFS afaik), but I wouldn't remove it at this juncture.
If removed, so should all the isQuery() associated code, but that's another PR.

I agree with @ghalliday comment though, the code above will treat the example file differently from the old code.
i.e. It will think ~file::ip::one::>two is an external file, when it did not before.

This example should fail on set, something similar can be a test case in the unittests.